code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -398,6 +398,13 @@
 			M.reagents.remove_reagent(R.type,1)
 	..()
 
+/datum/reagent/medicine/charcoal/reaction_mob(mob/living/carbon/M, method=TOUCH, reac_volume)
+	if(method == INGEST)
+		return
+	M.reagents.remove_reagent(/datum/reagent/medicine/charcoal/, reac_volume) //We really should not be injecting an insoluble granular material.
+	M.reagents.add_reagent(/datum/reagent/carbon, reac_volume) // It's pores would get clogged with gunk anyway.
```

Comment:
"its" pores, not "it's"

reeeeeeeee

Reply:
night coding, i will take care of it.

Reply:
Both are valid but the apostrophe-less version is better because it can't confused with the shorthand for "it is".

Reply:
fixed